statsd.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+var TXStatsD = require('node-txstatsd');
+
+// StatsD wrapper
+function StatsD(statsdHost, statsdPort) {
+    this.statsd = new TXStatsD({
+        host: statsdHost,
+        port: statsdPort,
+        prefix: 'restbase.routes.',
```

Comment:
`prefix` should be parametrised so that each service can have its prefix. Idem for `suffix` one line below.